matomo/values.yaml

```
@@ -21,13 +21,55 @@ service:
   annotations: {}
   labels: {}
 
+persistence:
+  enabled: true
+
+  enabled: true
+  annotations: {}
+  accessMode: ReadWriteOnce
+  existingClaim: ""
+  ## database data Persistent Volume Storage Class
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  ##
+  # storageClass: "-"
+  size: 2Gi
+  
+  data:
+    mountPath: "/data"
+    subPath: ""
```

Unnötige Einordnung unter `data`.
```suggestion
  mountPath: "/data"
  subPath: ""
```